app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
+
+  has_many :groups, class_name: 'Group', foreign_key: 'user_id'
+  has_many :expenses, class_name: 'Expense', foreign_key: 'user_id'
+
+  validates :name, presence: true, length: { in: 2..300 }
```

Comment:
- [OPTIONAL] Great job adding user name validation 👏🏻 Let's now validate all user input (password, ...) to make sure that anyone with bad intentions cannot compromise their app.